content/en/docs/reference/using-api/server-side-apply.md

```diff
@@ -198,81 +256,82 @@ data:
+| `//+mapType` | `x-kubernetes-map-type` | `atomic`/`granular` | Applicable to maps. `atomic` means that the map can only be entirely replaced by a single manager. `granular` means that the map supports separate managers updating individual fields. |
+| `//+structType` | `x-kubernetes-map-type` | `atomic`/`granular` | Applicable to structs; otherwise same usage and OpenAPI annotation as `//+mapType`.|
 
 If `listType` is missing, the API server interprets a
 `patchMergeStrategy=merge` marker as a `listType=map` and the
 corresponding `patchMergeKey` marker as a `listMapKey`.
 
 The `atomic` list type is recursive.
 
-These markers are specified as comments and don't have to be repeated as
-field tags.
+(In the [Go](https://go.dev/) code for Kubernetes, these markers are specified as
+comments and code authors need not repeat them as field tags).
 
 ### Compatibility across topology changes
 
-On rare occurrences, a CRD or built-in type author may want to change the
-specific topology of a field in their resource without incrementing its
-version. Changing the topology of types, by upgrading the cluster or
-updating the CRD, has different consequences when updating existing
-objects. There are two categories of changes: when a field goes from
-`map`/`set`/`granular` to `atomic` and the other way around.
+On rare occurrences, the author for a CustomResourceDefinition (CRD) or built-in
+may want to change the specific topology of a field in their resource,
+without incrementing its API version. Changing the topology of types,
+by upgrading the cluster or updating the CRD, has different consequences when
+updating existing objects. There are two categories of changes: when a field goes from
+`map`/`set`/`granular` to `atomic`, and the other way around.
 
 When the `listType`, `mapType`, or `structType` changes from
 `map`/`set`/`granular` to `atomic`, the whole list, map, or struct of
 existing objects will end-up being owned by actors who owned an element
 of these types. This means that any further change to these objects
 would cause a conflict.
 
-When a list, map, or struct changes from `atomic` to
-`map`/`set`/`granular`, the API server won't be able to infer the new
+When a `listType`, `mapType`, or `structType` changes from `atomic` to
+`map`/`set`/`granular`, the API server is unable to infer the new
 ownership of these fields. Because of that, no conflict will be produced
 when objects have these fields updated. For that reason, it is not
 recommended to change a type from `atomic` to `map`/`set`/`granular`.
 
 Take for example, the custom resource:
 
 ```yaml
+---
 apiVersion: example.com/v1
 kind: Foo
 metadata:
   name: foo-sample
   managedFields:
-  - manager: manager-one
+  - manager: "manager-one"
```

Comment:
nit: quotes?

Reply:
I don't personally have an opinion on that.

Reply:
I added the quotes to make it a bit more clear that a manager name is an opaque (?) string. Even though YAML lets you leave it out.